### ISSUE TITLE
In the inner network environment, kafak-ui become blank page, how to solve this problem?

### ISSUE THREAD

**Issue by anarckk** [provectus/kafka-ui#3546]
![Snipaste_2023-03-24_11-25-34](https://user-images.githubusercontent.com/27038948/227417229-1ce18414-2f2d-44a4-aaa4-8a54895455b9.png)

I blocked `https://api.github.com/repos/provectus/kafka-ui/releases/latest`, and repeatted the problem like inner network environment.

**Comment by anarckk** [comment]
it's my docker-compose.yaml, can I just add some environment variable to solve this problem?
```
  kafka-ui:
    image: provectuslabs/kafka-ui:latest
    container_name: kafka-ui
    ports:
      - 20116:8080
    restart: unless-stopped
    environment:
      - TZ=Asia/Shanghai
      - KAFKA_CLUSTERS_0_NAME=local
      - KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS=kafka:9093
```

**Comment by anarckk** [comment]
I found solution.

**Comment by ClockQ** [comment]
> I found solution.

Hi bro, can you tell me the solution?

**Comment by ClockQ** [comment]
> I found solution.

@anarckk can you tell me the solution?

**Comment by Haarolean** [comment]
@ClockQ if you're in an "air gap" environment, we have some fixes on the way (#2753)

**Comment by ClockQ** [comment]
@anarckk 谢谢大佬，您的方案可以让我优化当前很不友好的方案，非常感谢您！
